research-prototype/Programs/Source/vectorized.mpc

```diff
@@ -0,0 +1,320 @@
+"""
```

Comment:
we incorporated all the vectorization into `ipae2e.mpc`. I don't think it makes sense to have two different implementations in the repo.